tests/test_generator.py

```diff
@@ -0,0 +1,106 @@
+from unittest.mock import MagicMock, patch, call
+
+from application import generators as GENERATORS
+
+
+@patch.object(GENERATORS, 'ImageDraw')
+@patch.object(GENERATORS, 'ImageFont')
+@patch.object(GENERATORS, 'Image')
+def test_generate_random_image(image_func, imagefont_func, imagedraw_func):
```

Comment:
Czy tak to patchować. Czytałem że odpowiednikiem tego jest jeszcze to że jako argument funkcji z testem możemy dać mock, a wewnątrz funkcji mock.patch.object(), następnie ustawić atrybut return_value stworzonego mocka.